subprojects/build.gradle.kts

```diff
@@ -18,10 +20,12 @@ buildscript {
 }
 
 plugins {
+    base
```

Comment:
What is the base?

Reply:
https://docs.gradle.org/current/userguide/base_plugin.html